task/build-vm-image/0.1/build-vm-image.yaml

```diff
@@ -0,0 +1,237 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: build-vm-image
+spec:
+  params:
+    - description: The platform to build on
+      name: PLATFORM
+      type: string
+    - name: OUTPUT_IMAGE
+      type: string
+      description: The output manifest list that points to the OCI artifact of the zipped ZM image
```

Comment:
What's `ZM`? Id it a typo? If not, could you expand it in the description?

Reply:
Fixed in https://github.com/ralphbean/build-definitions/commit/71b679b4ada27216a62b233434601064339f714f.